docs/logging.md

```diff
@@ -28,6 +28,58 @@ The number is 1
 ```
 When the last `CHECK` fails in the "Bar" test case, then only one message will be printed: `Test case start`.
 
+## Logging without local scope
+
+`UNSCOPED_INFO` is similar to `INFO` with two key differences:
+
+- Lifetime of unscoped info is not tied to its own scope.
+- Unscoped info can be reported by the first following assertion only, regardless of the result of that assertion.
+
+This macro can be used for reporting information from helper functions or inner scopes because lifetime of `UNSCOPED_INFO` is limited by the assertion following whereas lifetime of `INFO` is limited by its own scope. An example:
```

Comment:
Split this sentence into multiple lines. Lifetime of `UNSCOPED_INFO` is not limited just by the following assertion, but also section end and test case end. 
`is limited by the assertion following` should be 'is limited by the following assertion`

Reply:
Done.